UCLALibrary/bucketeer
# [SERV-532] Retry Slack API requests on failure

To test:

1. Run `mvn clean test -Dtest=SlackMessageVerticleTest`; observe test success after around 2 seconds (give or take)
2. Disconnect your computer running Maven from the internet **and** repeat step 1; observe test failure after around 30 seconds
3. Repeat step 1 **and** around 15 seconds into the test execution, reconnect your computer to the internet; observe test success after around 20 seconds
